force-app/main/default/triggers/CaseTrigger.trigger-meta.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version="1.0" encoding="UTF-8"?>
 <ApexTrigger xmlns="http://soap.sforce.com/2006/04/metadata">
-  <apiVersion>60.0</apiVersion>
-  <status>Active</status>
-</ApexTrigger>
\ No newline at end of file
+    <apiVersion>57.0</apiVersion>
```

Comment:
@deepakmulamalla Please advise what created the requirement for this api rollback.

Reply:
@cgijeffolsen, I've reverted the changes back to Api version v60.